src/overlays/actors/ovl_En_Toryo/z_en_toryo.c

```diff
@@ -233,11 +233,11 @@ s32 func_80B205CC(EnToryo* this, PlayState* play) {
     return ret;
 }
 
-u32 func_80B20634(EnToryo* this, PlayState* play) {
+u32 EnToryo_OfferItem(EnToryo* this, PlayState* play) {
     u32 ret;
```

Comment:
we recently started naming the "get item" functions, and have potentially settled on using "offer" for offering the get item exchange. considering this doesnt deal with that interaction at all, and is instead more about getting text ids, we should maybe reconsider